README.md

```diff
@@ -27,13 +38,18 @@ To generate a configuation file template, run the command below in your terminal
 cargo run -- print-config
 ```
 
-Next, create a `toml` file in the root of the application, replacing the default keys in the template displayed in your terminal with your configuration. Make sure to confirm that the token info in your configuration file matches the deployed cellar contract.
+Next, create a `toml` file in the root of the application, replacing the default keys in the template displayed in your terminal with your configuration. Make sure to confirm that the token info in your configuration file matches the deployed Cellar contract.
 
 You can create keys or import keys. To create keys, run the command below:
 
 ```
 cargo run -- -c [your_config_file_name.toml] keys add [key_name]
 ```
+If you already have a key, you can import it with the command below:
+
+```
+cargo run -- -c [your_config_file_name.toml] keys import [key_name]
```

Comment:
I didn't notice it earlier, but the keys commands will need to specify `cosmos` or `eth`. They also still need to be moved from `cargo run --` to `cargo run --bin steward` because of the workspace change.

Reply:
done!